archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -120,11 +120,14 @@
 				<version>${compiler-plugin.version}</version>
 			</plugin>
 			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-war-plugin</artifactId>
 				<version>${war-plugin.version}</version>
+#if (${profile} == 'core')
 				<configuration>
 					<failOnMissingWebXml>false</failOnMissingWebXml>
```

Comment:
Since the lowest version we support is Jakarta EE 8, looks like we could just remove failOnMissingWebXml altogether. That would keep this code simple.

Reply:
I tried, but it doesn't work for core profile. There's no Servlet API on the classpath for core profile, so Maven War plugin expects by default that a web.xml is present and fails. See https://maven.apache.org/plugins/maven-war-plugin/war-mojo.html#failOnMissingWebXml

Reply:
OK, this is fine. Hopefully the Maven plugin will adapt in the future.